aten/src/ATen/native/native_functions.yaml

```diff
@@ -720,6 +720,17 @@
     CPU: _gesv_helper_cpu
     CUDA: _gesv_helper_cuda
 
+- func: _gesv_single(Tensor self, Tensor A) -> (Tensor, Tensor)
```

Comment:
this doesn't need two implementations, right?

Reply:
I meant that `_gesv_single_cpu` and `_gesv_single_cuda` can be merged together. They only differ by the `_gesv_single_out_cpu` vs. `_gesv_single_out_cuda` call, but you can use `_gesv_single_out` to dispatch to correct version.